src/lib.rs

```diff
@@ -207,11 +207,23 @@ pub fn generate(mut args: Args) -> Result<()> {
 
 fn create_git(args: Args, name: &ProjectName) -> Result<()> {
     let force = args.force;
+    use itertools::*;
     let template_values = args
         .template_values_file
-        .as_ref()
-        .map(|p| Path::new(p))
-        .map_or(Ok(Default::default()), |path| get_config_file_values(path))?;
+        .iter()
```

Comment:
the `create_git` function has grew a lot, could you please refactor this part that does now the template value collection into an own function something like `collect_template_values_from_files` to reduce the responsibilities of this function. 

You could then also add a very small unit test for the function.